build/pipelines/pipeline.properties

```diff
@@ -0,0 +1,24 @@
+# Visual Studio solution path
+solutionPath=SFTPAdapter.sln
+
+# Configurations to build and execute (comma separated list)
+projectConfigurations=Release
+
+# Application projects that are to be compiled (comma separated list)
+buildProjects=SFTPClientCLI
+
+# Test projects to be executed
+# testProjects=SFTPClientTest
+
+# Folder where artifacts are generated
+artifactsFolder=artifact
```

Comment:
We are not archiving any artifact during the build. If not needed, remove it from here.

Reply:
mmm well.. maybe the problem is that we are not archiving anything...
We should archive the CLI so that the test team can do manual test if needed, I think. 
Thus, let's leave this here until the CLI is done.